lib/measured/unit_system_builder.rb

```diff
@@ -26,27 +27,27 @@ def build
   private
 
   SI_PREFIXES = [
-    ["y", "yocto", -24],
-    ["z", "zepto", -21],
-    ["a", "atto", -18],
-    ["f", "femto", -15],
-    ["p", "pico", -12],
-    ["n", "nano", -9],
-    ["μ", "micro", -6],
-    ["m", "milli", -3],
-    ["c", "centi", -2],
-    ["d", "deci", -1],
-    ["da", "deca", 1],
-    ["h", "hecto", 2],
-    ["k", "kilo", 3],
-    ["M", "mega", 6],
-    ["G", "giga", 9],
-    ["T", "tera", 12],
-    ["P", "peta", 15],
-    ["E", "exa", 18],
-    ["Z", "zetta", 21],
-    ["Y", "yotta", 24]
-  ]
+    ["y", "yocto", -24].freeze,
+    ["z", "zepto", -21].freeze,
+    ["a", "atto", -18].freeze,
+    ["f", "femto", -15].freeze,
+    ["p", "pico", -12].freeze,
+    ["n", "nano", -9].freeze,
+    ["μ", "micro", -6].freeze,
+    ["m", "milli", -3].freeze,
+    ["c", "centi", -2].freeze,
+    ["d", "deci", -1].freeze,
+    ["da", "deca", 1].freeze,
+    ["h", "hecto", 2].freeze,
+    ["k", "kilo", 3].freeze,
+    ["M", "mega", 6].freeze,
+    ["G", "giga", 9].freeze,
+    ["T", "tera", 12].freeze,
+    ["P", "peta", 15].freeze,
+    ["E", "exa", 18].freeze,
+    ["Z", "zetta", 21].freeze,
+    ["Y", "yotta", 24].freeze,
+  ].freeze
```

Comment:
Nitpick: you could use `map(&:freeze).freeze`

Reply:
Oh I like that.